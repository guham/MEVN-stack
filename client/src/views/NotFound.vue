<template>
  <div class="text-center text-4xl font-bold">
    NotFound
  </div>
</template>

<script>
export default {
  name: 'NotFound',
  metaInfo: {
    title: '404 Page not found',
  },
  data: () => ({}),
};
</script>
