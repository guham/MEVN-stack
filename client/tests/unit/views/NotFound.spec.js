import NotFound from '@/views/NotFound';
import factory from '../factory';

describe('NotFound.vue', () => {
  test('has the expected html structure', () => {
    const wrapper = factory(NotFound);
    const template = wrapper.html();
    expect(template).toMatchSnapshot();
  });
});
