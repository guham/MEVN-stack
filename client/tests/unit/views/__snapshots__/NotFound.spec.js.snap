// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound.vue has the expected html structure 1`] = `
<div class="text-center text-4xl font-bold">
  NotFound
</div>
`;
